### DISCUSSION TITLE
CSS loading delay with Gatsby and Mantine

### DISCUSSION THREAD

**Discussion by xxbxxqxx** [mantinedev/mantine#3270] [Q&A]
### What package has an issue

@mantine/core

### Describe the bug

Hi I'm making a Gatsby website with Mantine.

When I access the page, naked HTML shows up and Styles by Mantine(I use Component and `createStyles()`) is loaded after a few seconds(some self-writing styles like layout.css can be seen in the first load, though).

it doesn't happen with `gatsby develop` but with `gatsby build` (or production build).

Related packages here:
```
  "dependencies": {
    "@emotion/react": "^11.10.5",
    "@emotion/server": "^11.10.0",
    "@mantine/core": "^5.9.5",
    "@mantine/ds": "^5.9.5",
    "@mantine/hooks": "^5.9.5",
    "@tabler/icons": "^1.118.0",
    "babel-plugin-styled-components": "^2.0.7",
    "gatsby": "^5.3.3",
    "gatsby-plugin-canonical-urls": "^5.3.0",
    "gatsby-plugin-google-tagmanager": "^5.3.1",
    "gatsby-plugin-manifest": "^5.3.1",
    "gatsby-plugin-mantine": "^5.0.0",
    "gatsby-plugin-sass": "^6.3.1",
    "gatsby-plugin-styled-components": "^6.3.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "styled-components": "^5.3.6"
  },
```

### What version of @mantine/hooks page do you have in package.json?

5.9.5

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
Make sure you've followed getting started guide. If you have issue check Gatsby template provided in documentation. You can also browse source code of Mantine documentation – it is built with Gatsby as well.
